zt_backend/models/components/list.py

```diff
@@ -0,0 +1,128 @@
+from pydantic import Field
+from typing import List, Optional, Union
+from zt_backend.models.components.zt_component import ZTComponent
+
+
+class ListComponent(ZTComponent):
```

Comment:
This should just be called "List". If that was causing issues let me know

Reply:
We had to name the component ListComponent since List was throwing an error (since it imports List from one of the inbuilt packages)